source/includes/_endpoint-search.md

```diff
@@ -0,0 +1,236 @@
+Performing a search with an invalid field will pinpoint the invalid one and provide the list of accepted values.
+
+Providing a Scope:
+
+`scope` parameter must be a JSON array containing the name of the fields that should be used to resolve the query. 
+Much like `projection` paramter this one accepts a list of fields with their namespace (`datasets` or `variables`). T
+he provided query will be looked up only in the specified fields if a `scope` is provided. A special field name `*`
+is accepted to specify that default fields should be looked for a specific namespace. 
+A scope like `datasets.name, variables.*` will search the query in the default variable fields and in dataset name.
+
+Grouping:
+
+Default grouping is ``datasets`` which will enable searching in dataset data and its variables. The returned entries
+in "datasets" are datasets that match the query or contain a variable that matches it. 
+Search results are limited to 1000 variables per dataset when grouping per dataset.
+
+Switching to ``variables`` grouping makes the search only look in variables.
+Note that a "datasets" field is still returned, the entries there are the datasets the matching variables are part of,
+*not* datasets that match the query. This is done to allow providing dataset details for a variable without the need for 
+a second call to fetch the dataset info.
+
+Allowable filter parameters:
+
+Parameter         | Type             | Description
+------------------|------------------|-------------------------------------------------
+dataset_ids       | array of strings | limit results to particular dataset_ids or urls (user must have read access to that dataset)
+team              | string           | url or id of the team to limit results (user must have read access to the team)
+project           | string           | url or id of the project to limit results (user must have access to the project)
+user              | string           | url or id of the user that has read access to the datasets to limit results (user must match with the provided one)
+owner             | string           | url or id of the dataset owner to limit results
+start_date        | array of strings | array of `[begin, end]` range of values in ISO8601 format. Provide same for exact matching.
+end_date          | array of strings | array of `[begin, end]` range of values in ISO8601 format. Provide same for exact matching.
+modification_time | array of strings | array of `[begin, end]` range of values in ISO8601 format. Provide same for exact matching.
+creation_time     | array of strings | array of `[begin, end]` range of values in ISO8601 format. Provide same for exact matching.
+
+<aside class="notice">
+The query string can only be alpha-numeric characters (including underscores) logical operators are not allowed at this time.
+</aside>
+
+
+##### Fields Searched
+
+Here is a list of the fields that are searched by the Crunch search endpoint
+
```

Comment:
i think for ease of use, add a column to define whether a field is a variable or dataset field.  Also, remove Post-Filter.